sirepo/template/webcon.py

```diff
@@ -356,6 +356,205 @@ def get_settings_report(run_dir, data):
 #    raise RuntimeError('{}: unknown simulation frame model'.format(data['modelName']))
 
 
+# use jinja?
+def jupyter_notebook_for_model(data, model):
+    # notebooks are JSON files
+    from sirepo.template.template_common import JupyterNotebook
+    nb = JupyterNotebook(SIM_TYPE, data)
+    nb.add_imports(
+        {
+            'numpy': []
+        }
+    )
+
+    nb.add_markdown_cell(['## Function definitions'])
+    nb.add_code_cell([
+        template_common.render_jinja(SIM_TYPE, {}, name='common.py')
+    ])
+    nb.add_code_cell([
+        template_common.render_jinja(SIM_TYPE, {}, name='analysis.py')
+    ])
+
+    data_var = 'data'
+    import csv
+    with open(str(_SIM_DATA.lib_file_abspath(_analysis_data_path(data)))) as f:
+        reader = csv.reader(f)
+        i = 0
```

Comment:
4c55b38

This kills the final comma if it exists, might as well be correct.  It also leverages an existing check for a header instead of assuming the 1st line is one.

I kept the original code structure (in a new function) because it's easier to do an empty array